scripts/download.py

```diff
@@ -7,11 +7,28 @@
 from typing import Optional
 
 from requests.exceptions import HTTPError
+import subprocess
 
+def download_tinyllamas(repo_id: str, local_dir: str) -> None:
+    try:
+        model_name = repo_id.split("/")[-1]
+        # Download model weight
+        weight_url = "https://huggingface.co/karpathy/tinyllamas/resolve/main/" + model_name + ".pt"
+        weight_dst_path = os.path.join(local_dir, "model.pth")
+        subprocess.run(["wget", weight_url, "-O", weight_dst_path], check=True)
+        # Download tokenizer model
+        tokenizer_url = "https://github.com/karpathy/llama2.c/blob/master/tokenizer.model"
```

Comment:
Should be https://github.com/karpathy/llama2.c/raw/master/tokenizer.model, as blob downloads an HTML document.

Reply:
Thanks, corrected!